fix(core): wrong zoom, row and col definition in extent

zarov opened the pull request:
The zoom, row and col properties, when the CRS is tiled, were
incorrectly defined, thus setting any of these properties later
impossible.

autra commented on src/Core/Geographic/Extent.js:
```diff
@@ -34,16 +34,22 @@ function Extent(crs, ...values) {
+                {
+                    get: () => this._col,
+                    set: (c) => { this._col = c; },
+                });
```
This got me wondering: why don't we directly use property access? I mean these getters and setters don't add logic or checks of any sort...

zarov replied:
I think it is simply because those properties can be available only when the CRS is tiled. But we could indeed write `this.zoom = this._zoom`.

zarov replied:
I'm rewriting all the file in fact, wait for the update

zarov committed:
fix(core): simplify zoom, row and col in Extent

When the CRS is tiled, there was two systems for accessing the zoom, row
and col properties of an Extent. The private properties were removed in
favor of simpler public properties.

zarov commented:
I removed the "private" properties in favor of simple "public" ones.

peppsac reviewed (approved):
LGTM, thanks!
